src/Engine.cpp

```diff
@@ -607,11 +607,62 @@ static void opt_version() {
 		"...please don't sue us." << std::endl;
 }
 
-bool Engine::parseCommandLine(int argc, char *argv[]) {
+bool Engine::parseCommandLine(int argc, char* argv[])
+{
+
 	// variables for command-line flags
 	int optret;
 	std::vector<std::string> data_vec;
 
+
+
+#ifdef _WIN32
```

Comment:
Can you move this whole thing below, so it's only used if the user doesn't pass anything on the command line? Still want to have the option to use the command line to launch games quickly and avoid this dialog